spec/unit/module/controller_spec.rb

```diff
@@ -5,12 +5,12 @@
 RSpec.describe Vtk::Commands::Module::Controller do
   it 'executes `module controller` command successfully' do
     output = StringIO.new
-    name = 'foo'
-    options = {}
+    name = 'bar'
```

Comment:
Should we test the scenario where the module name is not provided. Looks like it's a required option

Reply:
Good call, I will add that in.